classes/Hook.php

```diff
@@ -126,10 +126,17 @@ class HookCore extends ObjectModel
     public function add($autodate = true, $null_values = false)
     {
         Cache::clean('hook_idsbyname');
+        Cache::clean('hook_active');
 
         return parent::add($autodate, $null_values);
     }
 
+    public function clearCache($all = false)
```

Comment:
```suggestion
    public function clearCache(bool $all = false): void
```

Reply:
Looks like this was on overzealous change:
Tests say:
> PHP Fatal error:  Uncaught Symfony\Component\Debug\Exception\ContextErrorException: Warning: Declaration of HookCore::clearCache(bool $all = false): bool should be compatible with ObjectModelCore::clearCache($all = false) in /home/runner/work/PrestaShop/PrestaShop/classes/Hook.php:1287